app/services/translation_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class TranslationService
+  def initialize(user=nil)
+    @user = user
+  end
+
+  def translate_for_post(post)
+    {
+      translation: "translation for post with ID: " + post.text.to_s 
+    }
+  end
+
+# Task 1:  Make request to DeeplSDervice
+# Task 2: Make front end
```

Comment:
Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).